content/en/tracing/trace_collection/compatibility/php.md

```diff
@@ -180,3 +180,4 @@ If the application invokes `pcntl_unshare(CLONE_NEWUSER);` and the tracer is ins
 [6]: https://www.php.net/manual/en/language.generators.overview.php
 [7]: https://www.php.net/manual/en/book.pcntl.php
 [8]: https://man7.org/linux/man-pages/man2/unshare.2.html
+[9]: /tracing/library_config/php/#environment-variable-configuration
```

Comment:
This page doesn't exist.

Reply:
oops. Fixed in 4589922e8a5c2959b815b301f5dbbfd082d5f7f1. Thanks